deploy/sre-prometheus/100-telemeter-client.PrometheusRule.yaml

```
@@ -11,8 +11,10 @@ spec:
   - name: sre-telemeter-client
     rules:
     - alert: MetricsClientSendFailingSRE
-      expr: absent(rate(metricsclient_request_send{status_code="200"}[5m]) > 0) or rate(metricsclient_request_send{status_code!="200"}[5m]) > 0
-      for: 15m
+      expr: 100 * sum by (job) (rate(metricsclient_request_send{status_code!="200"}[5m])) / sum by (job) (rate(metricsclient_request_send[5m])) > 14.4 * 10 and 100 * sum by (job) (rate(metricsclient_request_send{status_code!="200"}[1h])) / sum by (job) (rate(metricsclient_request_send[1h])) > 14.4 * 10
```

```suggestion
      expr: sum by (job) (rate(metricsclient_request_send{status_code!="200"}[5m])) / sum by (job) (rate(metricsclient_request_send[5m])) > (6 - (1-0.9000) and sum by (job) (rate(metricsclient_request_send{status_code!="200"}[1h])) / sum by (job) (rate(metricsclient_request_send[1h])) > (6 - (1-0.90000)
```

Upon really digging into this promQL, I found the following:

- telemeter client only sends metrics every 5 minutes. This was validated using `sum(metricsclient_request_send) - sum(metricsclient_request_send offset 10m)` returning a `value=2`. 
- Using a 14.4 burn rate on an SLO of 90% will never trigger as it requires 2% of the error budget in a single hour to be consumed. Because a 100% outage consumes only 1.4% of the budget in that hour, this alert could never fire.
- Updated the burn rate to use 6 which will actually trigger and allow reset to occur as a binary using the 5m windows
- Updated the pass the SLO measurement rather then the error budget itself. 

With this, the 5m window becomes a binary switch allowing fast reset and the 1h window allows for up to 60% (7+ requests) of requests to fail for the hour before triggering.

---

Worth to note that this is based on the 90% SLO of telemeter itself